verify/Dockerfile

```diff
@@ -5,11 +5,13 @@ RUN dnf -y install git yum-utils npm rpm-build curl libvirt-client openssl rpmde
          rpm-build krb5-workstation python-lxml expect selinux-policy-devel tar bzip2 python bind-utils \
          nc gnupg freetype fontconfig krb5-workstation expect psmisc mock libvirt \
          libvirt-python sudo python-libguestfs net-tools virt-install
+RUN dnf -y install kernel grubby libguestfs-xfs
```

Comment:
Good point. I thought this would be good to leverage layers so people could reuse the older layers, but I realized that with new package versions this layer will change anyway.